lib/miq_automation_engine/engine/miq_ae_engine/miq_ae_object.rb

```diff
@@ -854,5 +854,11 @@ def classify_value(value)
         return self[value]
       end
     end
+
+    def attribute_for_vmdb_object?(klass, value)
+      Kernel.const_defined?(klass) && value.to_i.nonzero?
+    rescue NameError
```

Comment:
Should  use `klass.safe_constantize` here which will return `nil` if the class does not exist.